docs/user/rstudio/ide/guide/code/code-sections.qmd

```diff
@@ -55,40 +68,51 @@ To insert a new code section you can use the **Code** -\> **Insert Section** com
      ### Section Three ############################# 
      ### Section Three ####
 
-Note that as illustrated above the line can start with any number of pound signs (#) so long as it ends with four or more `-`, `=`, or `#` characters.
+::: {callout-tip}
+
+## Code Section Levels
 
-To navigate between code sections you can use the **Jump To** menu available at the bottom of the editor:  
+As illustrated above, the line can start with any number of pound signs (#) so long as it ends with a space followed by four or more `-`, `=`, or `#` characters.
 
-![](images/plot-examples-navigation.png){fig-alt="A screenshot of the Jump To Menu, which is at the bottom of the Source Pane."}
+:::
 
-You can also navigate between code sections with the **Document Outline** on the **Source Pane** menu bar.  
+To navigate between code sections, use the **Jump To** menu available at the bottom of the editor:
+
+![](images/plot-examples-navigation.png){fig-alt="A screenshot of the Jump To Menu, which is at the bottom left of the Source Pane."}
+
+Code sections can also be navigated with the **Document Outline** on the **Source Pane** menu bar.
 
 ![](images/plot-examples-outline.png){fig-alt = "A screenshot of a .R file with code sections that can be navigated via the Document Outline"}
 
```

Comment:
`{fig-alt = "A screenshot of a .R file with code sections that can be navigated via the Document Outline"}` appears below the image in the html. I don't think this was intentional, but if so the **Jump To Menu** image above and **Quarto Screenshot** below also have the `fig-alt` set but the text isn't displayed in the html.

Reply:
Fixed!